repose-aggregator/components/filters/openstack-identity-v3-filter/src/test/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3APITest.scala

```diff
@@ -120,14 +121,14 @@ class OpenStackIdentityV3APITest extends FunSpec with BeforeAndAfterEach with Ma
     }
 
     it("should build a JSON auth token request with a string project ID") {
-      val mockServiceClientResponse = mock[ServiceClientResponse]
-
-      identityConfig.getOpenstackIdentityService.setProjectId("projectId")
-
+      val serviceClientResponse = new ServiceClientResponse(
+        SC_UNAUTHORIZED,
+        List.empty[Header].toArray,
```

Comment:
would `Array.empty` save you a step? Does it even exist for that matter

Reply:
Updated.